File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/view/interfaceview/SignInView.java
package com.c_store.zhiyazhang.cstoremanagement.view.interfaceview;

import com.c_store.zhiyazhang.cstoremanagement.bean.UserBean;

/**
 * Created by zhiya.zhang
 * on 2017/5/8 14:59.
 */

public interface SignInView {
    String getUID();

    String getPassword();

    void showLoading();

    void hideLoading();

    void toActivity(UserBean user);

    void showFailedError(String errorMessage);

    void saveUser(UserBean user);
}

File: CStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/view/interfaceview/SignInView.kt
package com.cstore.zhiyazhang.cstoremanagement.view.interfaceview

import com.cstore.zhiyazhang.cstoremanagement.bean.User

/**
 * Created by zhiya.zhang
 * on 2017/5/8 14:59.
 */

interface SignInView {
    /**
     * 得到输入的id
     */
    val uid: String

    /**
     * 得到输入的密码
     */
    val password: String

    /**
     * 保存用户信息
     */
    fun saveUser(user: User)
}
